enterprise/decorators.py

```diff
@@ -17,58 +15,6 @@
 from six.moves.urllib.parse import parse_qs, urlencode, urlparse, urlunparse  # pylint: disable=import-error
 
 
-def deprecated(extra):
```

Comment:
this isn't specific to the old consent stuff although it brought this up as being useful - i would argue we should keep this because i'm sure we'll be deprecating other stuff. unless there is another way we should be doing that going forward?

Reply:
Hmm, I think the decorators did help me remember what it was that needed to be deleted (that's why I put 'em in the first place -- and to bug us in the test logs into doing so). But I think now that it's unused, I'll have to add unit tests for coverage D: But no worries I'll do that :+1:

Reply:
@brittneyexline I've added these back and added unit tests.